operator/templates/service-monitor.yaml

```diff
@@ -17,5 +16,4 @@ spec:
   selector:
     matchLabels:
       kudo.dev/instance: {{ .Name }}
-      kudo.dev/servicemonitor: "true"
-{{ end }}
+      kudo.dev/servicemonitor: "true"
```

Comment:
Can you please keep the trailing newline? It's visual noise when it's getting added and removed.

Reply:
Well, the change here was the removal of `{{ end }}` in the last line, not the trailing new line, but i've added it nonetheless back in.